tensornetwork/backends/numpy/decompositions.py

```diff
@@ -26,7 +27,7 @@ def svd(
     max_truncation_error: Optional[float] = None,
     relative: Optional[bool] = False) -> Tuple[Tensor, Tensor, Tensor, Tensor]:
   """Computes the singular value decomposition (SVD) of a tensor.
-
+  
```

Comment:
There are a lot of places where you add whitespace like this. Could you remove this from this PR? Thanks